pkgs/top-level/all-packages.nix

```diff
@@ -19242,6 +19249,7 @@ in
   firefoxPackages = recurseIntoAttrs (callPackage ../applications/networking/browsers/firefox/packages.nix {
     callPackage = pkgs.newScope {
       inherit (gnome2) libIDL;
+      inherit (rustPackages_1_44) cargo rustc;
```

Comment:
I am not a huge fan of updating cargo and rustc for all firefox packages. Why not just pull them in for those version of firefox where we need them like we do with `nss`, `nspr`, …?

Reply:
Idk why I did it that way. I did it the other way now. Seems like a no-op `firefox`. `firefox-esr-78` still gets rebuilt, because of the nss/nspr bump, `firefox-esr-68` shouldn't be touched by this.